quiche/include/quiche.h

```diff
@@ -630,6 +629,58 @@ enum quiche_h3_error {
     // The requested operation cannot be served over HTTP/3. Peer should retry
     // over HTTP/1.1.
     QUICHE_H3_ERR_VERSION_FALLBACK = -20,
+
+    // QUIC transport had no more work to do.
```

Comment:
Nit: it just occurred to me that we don't really need to duplicate the decriptions here, could just be `// See QUICHE_ERR_...` or something like that, so it's easier to keep everything in sync in case we change something in the future.

Reply:
good idea!